lib/account.rb

```diff
@@ -0,0 +1,26 @@
+class Account
+
+attr_reader :balance, :transactions
+
+  def initialize(balance = 0)
+    @balance = balance
```

Comment:
Do you need to aggregate the balance? Or can you calculate it when it's required?